pkg/util/mount/mount_linux.go

```diff
@@ -331,26 +344,26 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 	options = append(options, "defaults")
 
 	// Run fsck on the disk to fix repairable issues
-	log.Printf("Checking for issues with fsck on disk: %s", source)
+	mounter.Logger.With("source", source).Info("Checking disk for issues using 'fsck'.")
 	args := []string{"-a", source}
 	cmd := mounter.Runner.Command("fsck", args...)
 	out, err := cmd.CombinedOutput()
 	if err != nil {
 		ee, isExitError := err.(utilexec.ExitError)
 		switch {
 		case err == utilexec.ErrExecutableNotFound:
-			log.Printf("'fsck' not found on system; continuing mount without running 'fsck'.")
+			mounter.Logger.Info("'fsck' not found on system; continuing mount without running 'fsck'.")
 		case isExitError && ee.ExitStatus() == fsckErrorsCorrected:
-			log.Printf("Device %s has errors which were corrected by fsck.", source)
+			mounter.Logger.With("source", source).Info("Device has errors that were corrected with 'fsck'.")
 		case isExitError && ee.ExitStatus() == fsckErrorsUncorrected:
-			return fmt.Errorf("'fsck' found errors on device %s but could not correct them: %s.", source, string(out))
```

Comment:
Probably don't want to change the behaviour here (`return`)

Reply:
Thanks for catching this. I completely missed it